.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/javascript-functions"]
+	path = submodules/javascript-functions
+	url = git@github.com:code-kern-ai/submodule-javascript-functions.git
```

Comment:
not what was meant with blank copy. I meant the files in the normal file structure added without the submodule integration via git. just a copy of the files so they are accessible